package-readmes/hmacAuthentication.md

```diff
@@ -93,7 +93,7 @@ To use the function below in your code, import it with the following syntax:
             response.body = "Incoming request is invalid";
```

Comment:
`response` is defined in the `try` block and so it's out of scope in the `catch` block